src/schema.tsx

```diff
@@ -84,7 +84,7 @@ export const typeDefs = gql`
   input CollectionRecord {
     id: ID!
     title: String!
-    bookmarks: [BookmarkRecord]
+    bookmarks: [BookmarkRecord!]!
```

Comment:
Same as above comment -- lmk if we want to allow an empty array to ever be returned from a CMS collection